client/src/totalRatings.jsx

```diff
@@ -6,14 +6,16 @@ import { faStar } from '@fortawesome/free-solid-svg-icons';
 const TotalRatings = ({ total, average }) => {
   const stars = [];
   for (let index = 1; index <= 5; index += 1) {
```

Comment:
Interesting that for stars array, the index starts with 1 (instead of 0)

Reply:
Yeah I'm using 1 because it is easier to read / understand when someone takes a look at it.